platform/troubleshoot/tsnet-connectivity.mdx

```diff
@@ -0,0 +1,190 @@
+---
+title: Resolve vCluster TSNet connection failures
+sidebar_label: Resolve TSNet connection failures
+description: Fix TSNet connectivity issues that prevent vClusters from showing as ready in the platform UI.
+---
+
+import Flow, { Step } from "@site/src/components/Flow";
+
+# Resolve vCluster TSNet connection failures
+
+The [TSNet (Tailscale Network)](https://tailscale.com/kb/1244/tsnet) connection failure prevents the vCluster Platform from properly marking virtual clusters as ready, even when they are running successfully. When this occurs, TSNet cannot establish a stable connection with the platform, which blocks communication between the vCluster on the connected cluster and the platform. As a result, the vCluster appears to be running in the host cluster but shows as it is not ready in the platform UI, which prevents proper management and monitoring.
```

Comment:
nit
```suggestion
The [TSNet (Tailscale Network)](https://tailscale.com/kb/1244/tsnet) connection failure prevents the vCluster Platform from properly marking virtual clusters as ready, even when they are running successfully. This occurs because TSNet cannot establish a stable connection with the platform, which blocks communication between the vCluster on the connected cluster and the platform. As a result, the vCluster appears to be running in the host cluster but shows as it is not ready in the platform UI, which prevents proper management and monitoring.
```